src/Compilers/VisualBasic/Portable/Binding/Binder_Lookup.vb

```diff
@@ -1644,9 +1700,10 @@ ExitForFor:
             End Function
 
             Private Shared Sub ClearLookupResultIfNotMethods(methodsOnly As Boolean, lookupResult As LookupResult)
-                If methodsOnly AndAlso
-                   lookupResult.HasSymbol AndAlso
-                   lookupResult.Symbols(0).Kind <> SymbolKind.Method Then
+                If lookupResult IsNot Nothing AndAlso
+                    methodsOnly AndAlso
```

Comment:
Nit: formatting is off by a space on these lines.

Reply:
No, I just verified this is the formatting applied by the IDE.